core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/ItemSelectionInputCheckerServiceSpec.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ItemSelectionInputCheckerService.
+ */
+
+describe('Checkbox input checker service', function() {
+  beforeEach(module('oppia'));
+
+  var AnswerGroupObjectFactory;
+  var mcics;
+
+  var customizationArgs = {
+    choices: {
+      value: [
+        '<p>Option 1</p>',
+        '<p>Option 2</p>',
+        '<p>Option 3</p>'
+      ]
+    }
+  };
+
+  beforeEach(inject(function($injector) {
+    AnswerGroupObjectFactory = $injector.get('AnswerGroupObjectFactory');
+    mcics = $injector.get('MultipleChoiceInputCheckerService');
```

Comment:
Why is this using the multiple-choice input checker service?

Reply:
Fixed!